ui/wallet_page.go

```diff
@@ -28,6 +28,18 @@ type optionMenuItem struct {
 	page   string
 }
 
+type addWalletMenuOption struct {
```

Comment:
This looks like it does the same thing with optionMenuItem, their only difference being action and page. The page field is used to set the page to change to when one of the optionMenuItems is clicked. The change page action can instead be done in action.